docs/source/mb_specification.rst

```diff
@@ -22,26 +22,33 @@ A MONAI Bundle is defined primarily as a directory with a set of specifically na
   ┃  ┗━ metadata.json
   ┣━ models
   ┃  ┣━ model.pt
-  ┃  ┗━ model.ts
+  ┃  ┣━ model.ts
+  ┃  ┗━ model.onnx
```

Comment:
Similar question as above, maybe put "*" for `model.ts` and `model.onnx` as they are optional?

Thanks.

Reply:
I think the texts described them as optional and the current PR improves the specification. If we identified new format that could be allowed as optional file, we can update this list. What do you think?

Reply:
I think that's fine, I added the stars anyway.